typescript/src/scss/main.scss

```diff
@@ -123,6 +134,45 @@ $low-opacity-gray: rgb(96 96 96 / 20%);
       color: red;
       font-size: smaller;
     }
+
+    input[type="checkbox"] {
+      height: 0;
+      width: 0;
+      visibility: hidden;
+    }
+
+    label {
+      cursor: pointer;
+      text-indent: -9999px;
+      width: 30px;
+      height: 16px;
+      background: #88cede;
```

Comment:
Can you change this to use a previously defined colour? (Guess this crept in when I copied my dev tools css!)